packages/devtools_app/lib/src/performance/performance_screen.dart

```diff
@@ -25,6 +25,9 @@ import '../theme.dart';
 import '../ui/vm_flag_widgets.dart';
 import 'performance_controller.dart';
 
+final performanceSearchFieldKey =
```

Comment:
why is this global key needed?

Reply:
ok. maybe add a TODO. there is probably an easier way to connect them together.

Reply:
done